tests/network/vmi_kubectl.go

```diff
@@ -0,0 +1,109 @@
+/*
+ * This file is part of the kubevirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2018 Red Hat, Inc.
+ *
+ */
+
+package network
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	v1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/client-go/kubecli"
+
+	"kubevirt.io/kubevirt/tests"
+	"kubevirt.io/kubevirt/tests/clientcmd"
+	"kubevirt.io/kubevirt/tests/libnet"
+	"kubevirt.io/kubevirt/tests/libvmi"
+	"kubevirt.io/kubevirt/tests/util"
+)
+
+const (
+	defaultPodNetworkName  = "default"
+	linuxBridgeIfaceName1  = "nic1"
+	linuxBridgeIfaceName2  = "nic2"
+	linuxBridgeNetworkName = "linux-br"
+	bridgeCNIType          = "bridge"
+	bridgeName             = "br10"
+
+	linuxBridgeNAD = `{"apiVersion":"k8s.cni.cncf.io/v1","kind":"NetworkAttachmentDefinition","metadata":{"name":"%s","namespace":"%s"},"spec":{"config":"{ \"cniVersion\": \"0.3.1\", \"name\": \"mynet\", \"plugins\": [{\"type\": \"%s\", \"bridge\": \"%s\", \"vlan\": %d, \"ipam\": {%s}, \"macspoofchk\": %t, \"mtu\": 1400},{\"type\": \"tuning\"}]}"}}`
+)
+
+var _ = SIGDescribe("Kubectl", func() {
+
+	var err error
+	var virtClient kubecli.KubevirtClient
+
+	getVmiHeader := []string{"NAME", "AGE", "PHASE", "IP", "NODENAME", "READY"}
+
+	createBridgeNetworkAttachmentDefinition := func(namespace, networkName string, bridgeCNIType string, bridgeName string,
+		vlan int, ipam string, macSpoofCheck bool) error {
+		bridgeNad := fmt.Sprintf(linuxBridgeNAD, networkName, namespace, bridgeCNIType, bridgeName, vlan, ipam, macSpoofCheck)
+		return createNetworkAttachmentDefinition(virtClient, networkName, namespace, bridgeNad)
+	}
+
+	BeforeEach(func() {
+		virtClient, err = kubecli.GetKubevirtClient()
+		util.PanicOnError(err)
+	})
+
+	Describe("kubectl get vmi test for primary and secondary interfaces", func() {
+		It("should verify vmi ip value match primary interface ip value", func() {
+			Expect(createBridgeNetworkAttachmentDefinition(util.NamespaceTestDefault, linuxBridgeNetworkName, bridgeCNIType,
+				bridgeName, 0, "", false)).To(Succeed())
+
+			vmi := libvmi.NewCirros(
```

Comment:
Does this test require a functional OS on the VM? Can it work with a diskless (without booting) VM?